backtester/eventtypes/signal/signal_test.go

```diff
@@ -30,3 +30,23 @@ func TestSetPrice(t *testing.T) {
 		t.Error("expected 1337")
 	}
 }
+
+func TestSetBuyLimit(t *testing.T) {
+	s := Signal{
+		BuyLimit: 10,
+	}
+	s.SetBuyLimit(20)
+	if s.GetBuyLimit() != 20 {
+		t.Error("expected 20")
```

Comment:
For ease of debugging can we also add the received amount to these errors please

Reply:
Just re-read this and I think I was unclear. Can you add s.GetBuyLimit in the error so theres a quick comparison of whats expected and what we receieve. something like: t.Error("expeted 20, received %v", ...) should suffice

Reply:
@MadCozBadd Done.